### PR TITLE
feat: add a tool to check the version of APISIX Dashboard for release

### LINKED ISSUE #961: feat: need a tool to check version before release
# Feature request

## Please describe your feature

we could refer to 
https://github.com/apache/apisix/blob/master/utils/check-version.sh

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
close #961

___
### New feature or improvement
- Describe the details and related test reports.

add a tool to check the version of APISIX Dashboard for release

### CODE REVIEW

File: utils/version-check.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+GREEN='\e[1;32m'
+NC='\e[0m'
+
+
+# doc: apisix $ver
+
+matched=`grep "apisix-dashboard-v[0-9][0-9.]*" -r docs/`
+expected=`grep "apisix-dashboard-v$ver" -r docs/`
+
+if [ "$matched" = "$expected" ]; then
```
Comment (starsz) [inline utils/version-check.sh:34]: What's this for? I cann't get your point.

File: utils/version-check.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    echo
+    echo "-----maybe wrong version-----"
+    echo "$matched"
+    exit 1
+fi
+
+matched=`grep "git clone -b v[0-9][0-9.]*" -r docs/`
+expected=`grep "git clone -b v$ver" -r docs/`
+
+if [ "$matched" = "$expected" ]; then
```
Comment (starsz) [inline utils/version-check.sh:47]: Ditto.

File: utils/version-check.sh [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+RED='\e[1;31m'
+green='\e[0;32m'
+GREEN='\e[1;32m'
+NC='\e[0m'
+
+
+# doc: apisix $ver
+
+matched=`grep "apisix-dashboard-v[0-9][0-9.]*" -r docs/`
+expected=`grep "apisix-dashboard-v$ver" -r docs/`
```
Comment (starsz) [inline utils/version-check.sh:32]: What's the difference between `matched` and `expected`?

File: Makefile [hunk 4]
```diff
@@ -103,8 +103,15 @@ endif
 	.actions/openwhisk-utilities/scancode/scanCode.py --config .actions/ASF-Release.cfg ./
 
 
+### version-check: Check Apache APISIX Dashboard version for release
+.PHONY: version-check
+version-check:
+	./utils/version-check.sh
```
Comment (imjoey) [inline Makefile:109]: Shall we need to pass a variable to run this script?

Comment (nic-chen) [reply Makefile:109]: good catch. thanks!

File: utils/version-check.sh (hunk 3 above)
Comment (nic-chen) [reply utils/version-check.sh:32]: List the content that matches `apisix-dashboard-v[0-9][0-9.]*` to see if it is the same as the specified version. If it is not the same, we can assume that the version number of such content has not been modified.

File: Makefile (hunk 4 above)
Comment (nic-chen) [reply Makefile]: fixed. please take a look again when you have time, thanks.

**Review by starsz — APPROVED** [review]

**Review by imjoey — APPROVED** [review]
/lgtm

thanks @nic-chen .

**Pull request merged**